swift/ios-shoppe-demo/Resources/Constants.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  Constants.swift
+//  ios-shoppe-demo
+//
+//  Created on 8/25/20.
+//  Copyright © 2020 FullStory All rights reserved.
+//
+
+
+import Foundation
+
+typealias nib = K.Nib
+typealias cellID = K.CellID
+
+/// Hard coded constants
+struct K {
+    struct Title {
+        static let orderCart = "Order Cart"
+        static let shoppe = "iOS Shoppe"
+    }
+
+    struct Nib {
+        static let productCollectionViewCell = "ProductCollectionViewCell"
+        static let cartTableViewCell = "CartTableViewCell"
+        static let cartQuant = "CartQuantityTableViewCell"
+        static let largeLabel = "LargeLabelTableViewCell"
+        static let checkBoxButton = "CheckBoxButton"
+        static let checkoutTableView = "CheckoutTableViewCell"
+        static let productSummaryCell = "ProductSummaryCell"
+        static let productSummaryTopCell = "ProductSummaryTopCell"
+        static let productSummaryUserDetailCell = "ProductSummaryUserDetailCell"
+        static let productSummaryTotalCost = "ProductSummaryTotalCost"
+        static let loginButtonCell = "LoginButtonViewCell"
+    }
+
+    struct CellID {
+        static let productView = "cell"
+        static let checkout = "checkout"
+        static let quantity = "quantity"
+        static let title = "title"
+        static let checkoutBox = "checkoutBox"
+        static let textField = "textField"
+        static let thankYou = "thankYou"
+        static let userDetails = "userDetails"
+        static let totalCell = "totalCell"
+        static let login = "login"
+    }
+
+    struct Image {
```

Comment:
I would take these out of the `K` struct and give them more descriptive names. Consider `TitleStrings`, `NibNames`, `CellIdentifiers`, and `ImageNames`.

These should be changed from `struct`s to `enum`s because an `enum` without cases cannot be instantiated.